Ginger/Ginger/Actions/ApiActionsConversion/ApiActionsConversionIntro.md

```diff
@@ -0,0 +1,5 @@
+﻿### 1- Why to convert web services actions to API Models?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4464228)